### DISCUSSION TITLE
Displaying shield badge for a pre-release nuget package with version <= latest stable version

### DISCUSSION THREAD

**Discussion by abhipsaMisra** [badges/shields#6943] [Q&A] [answered]
For the following scenario:

- latest stable - 1.25.0
- latest pre-release - 1.25.0-preview-001

Since, based on semver, 1.25.0 > 1.25.0-preview-001, this results in both of the below resolving to 1.25.0:
[iothub-device-release]: https://img.shields.io/nuget/v/Microsoft.Azure.Devices.Client.svg?style=plastic
[iothub-device-prerelease]: https://img.shields.io/nuget/vpre/Microsoft.Azure.Devices.Client.svg?style=plastic

Since I want [iothub-device-prerelease] to resolve to 1.25.0-preview-001 instead, is there a way to achieve this?

**Comment by chris48s** [comment, marked as answer]
Nuget always sorts versions by semver.
`/vpre` would only give you a different result if there's a _more recent_ pre-release.

So for example if your package had `1.25.0` and `1.26.0-preview-001` then
- `/v` would show `1.25.0` (latest stable version) and
- `/vpre` would give you `1.26.0-preview-001` (latest version including pre-releases)
